.github/workflows/docker-image.yml

```diff
@@ -1,4 +1,20 @@
 name: Publish Docker image
+#
```

Comment:
Shall we put the license in the file header?

Reply:
do you mean to move to L1? that's invalid for GA

Reply:
It works fine for spark, Is anything wrong?

https://github.com/apache/spark/blob/1112240ffc5456778124e43ec75aae5d8cb7121f/.github/workflows/build_and_test.yml#L1-L18

Reply:
Ok, thanks for the inputs. I might make some mistake before